README.md

```diff
@@ -248,28 +252,30 @@ The table below compares the performance (measured in terms of average elapsed t
 
 |  distance  |  loop  |  pairwise  |  gain  |
 |----------- | -------| ----------| -------|
-| SqEuclidean | 0.015116s |  0.000192s | **78.7747** |
-| Euclidean | 0.015565s |  0.000390s | 39.8829 |
-| Cityblock | 0.015048s |  0.001400s | 10.7469 |
-| Chebyshev | 0.023325s |  0.010921s |  2.1358 |
-| Minkowski | 0.143427s |  0.121050s |  1.1849 |
-| Hamming | 0.015191s |  0.001334s | 11.3856 |
-| CosineDist | 0.016688s |  0.000393s | **42.5158** |
-| CorrDist | 0.029024s |  0.000435s | **66.7043** |
-| ChiSqDist | 0.026035s |  0.012194s |  2.1351 |
-| KLDivergence | 0.115800s |  0.086968s |  1.3315 |
-| RenyiDivergence | 0.055551s |  0.029628s |  1.8749 |
-| RenyiDivergence | 0.205270s |  0.163031s |  1.2591 |
-| JSDivergence | 0.165078s |  0.148902s |  1.1086 |
-| BhattacharyyaDist | 0.035493s |  0.022429s |  1.5824 |
-| HellingerDist | 0.035028s |  0.021867s |  1.6019 |
-| WeightedSqEuclidean | 0.016330s |  0.000276s | **59.2117** |
-| WeightedEuclidean | 0.016600s |  0.000508s | **32.6478** |
-| WeightedCityblock | 0.015604s |  0.001816s |  8.5913 |
-| WeightedMinkowski | 0.159052s |  0.128427s |  1.2385 |
-| WeightedHamming | 0.015212s |  0.001634s |  9.3110 |
-| SqMahalanobis | 0.607881s |  0.000365s | **1665.3228** |
-| Mahalanobis | 0.623032s |  0.000604s | **1031.9581** |
-| BrayCurtis | 0.015843s |  0.002273s |  6.9695 |
+| SqEuclidean | 0.010946s |  0.001245s |  8.7912 |
+| Euclidean | 0.011314s |  0.001328s |  8.5189 |
+| PeriodicEuclidean | 0.028662s |  0.018289s |  1.5672 |
+| Cityblock | 0.011112s |  0.001841s |  6.0349 |
+| TotalVariation | 0.011052s |  0.001906s |  5.7993 |
+| Chebyshev | 0.022198s |  0.012849s |  1.7276 |
+| Minkowski | 0.116707s |  0.104321s |  1.1187 |
+| Hamming | 0.010888s |  0.001733s |  6.2837 |
+| CosineDist | 0.015977s |  0.001278s | 12.5042 |
+| CorrDist | 0.026070s |  0.001338s | 19.4795 |
+| ChiSqDist | 0.013399s |  0.004151s |  3.2276 |
+| KLDivergence | 0.075753s |  0.062430s |  1.2134 |
+| RenyiDivergence | 0.041190s |  0.027761s |  1.4838 |
+| RenyiDivergence | 0.127884s |  0.114440s |  1.1175 |
+| JSDivergence | 0.114959s |  0.106575s |  1.0787 |
+| BhattacharyyaDist | 0.019642s |  0.010409s |  1.8870 |
+| HellingerDist | 0.019028s |  0.009799s |  1.9418 |
+| WeightedSqEuclidean | 0.011327s |  0.001252s |  9.0478 |
+| WeightedEuclidean | 0.011520s |  0.001338s |  8.6124 |
+| WeightedCityblock | 0.011308s |  0.002538s |  4.4547 |
+| WeightedMinkowski | 0.121861s |  0.105896s |  1.1508 |
+| WeightedHamming | 0.011249s |  0.002448s |  4.5945 |
+| SqMahalanobis | 0.189282s |  0.001452s | 130.3337 |
```

Comment:
I'm a bit surprised by the relative slowdowns in the `pairwise` method here (and in some other places). Perhaps that is due to the CPU used here being weaker and not gaining as much from using BLAS.

Reply:
Hm, I overlooked that. Those BLAS-optimized `pairwise` methods are not touched by my PR or by the `parameter` trait, are they? Can we run the benchmark script on the original reference computer?

Reply:
I'll try run it on my computer 9900k and see what I get.

Reply:
Donnu what went wrong. I'm getting much better values now, even better gains than were documented before in the README. I'll push the new tables in a minute.